apps/minaintyg/src/components/Layout/Layout.tsx

```diff
@@ -7,7 +8,8 @@ export function Layout() {
     <div className="flex min-h-screen flex-col">
       <LayoutHeader />
       <main className="flex-1">
-        <div className="ids-content m-auto max-w-7xl py-10 px-2.5">
+        <div className="ids-content m-auto max-w-7xl px-2.5 py-5">
```

Comment:
This can be a general component to get a more readable name what the styling creates

Reply:
The general component is `Layout`, the element is not used anywhere else, so there is currently no need to name it. This could change in the future, but would be easy to change then.